actions/image.search.by.regex.yaml

```diff
@@ -0,0 +1,102 @@
+---
+description: Search for image property using regex pattern, or not matching given value(s)
+enabled: true
+entry_point: src/openstack_actions.py
+name: image.search.by.regex
+parameters:
+  timeout:
+    default: 5400
+  lib_entry_point:
+    default: workflows.search_by_regex.search_by_regex
+    immutable: true
+    type: string
+  query_type:
+    default: ImageQuery
+    immutable: true
+    type: string
+  cloud_account:
+    description: The clouds.yaml account to use whilst performing this action
+    required: true
+    type: string
+    default: "dev"
+    enum:
+      - "dev"
+      - "prod"
+  properties_to_select:
+    default:
+      - "image_name"
```

Comment:
Should this default to selecting all properties like the other actions

Reply:
Yes have changed this thank you